lib/python/src/bailo/helper/model.py

```diff
@@ -111,6 +111,98 @@ def from_id(cls, client: Client, model_id: str) -> Model:
 
         return model
 
+    @classmethod
+    def from_mlflow(
+        cls,
+        client: Client,
+        mlflow_uri: str,
+        team_id: str,
+        name: str,
+        schema_id: str | None = None,
+        version: str | None = None,
+        files: bool = True,
+        visibility: ModelVisibility | None = None,
+    ) -> Model:
+        """Import an MLFlow Model into Bailo.
+
+        :param client: A client object used to interact with Bailo
+        :param mlflow_uri: MLFlow server URI
+        :param team_id: A unique team ID
+        :param name: Name of model (on MLFlow). Same name will be used on Bailo
+        :param schema_id: A unique schema ID, only required when files is True, defaults to None
+        :param version: Specific MLFlow model version to import, defaults to None
+        :param files: Import files?, defaults to True
+        :param visibility: Visibility of model on Bailo, using ModelVisibility enum (e.g Public or Private), defaults to None
+        :return: A model object
+        """
+        if ml_flow:
```

Comment:
Invert these checks so that the error message is location by the if statement.  E.g. instead of:

```
if x:
  if y:
    if z:
      // do a
    else:
      throw 'z'
  else:
    throw 'y'
else:
  throw 'x'
```

Instead:

```
if not x:
  throw 'x'

if not y:
  throw 'y'

if not z:
  throw 'z'

// do a
```

At the moment I have to scroll through ~30 lines of code to see where the source is.

Reply:
Done